### ISSUE TITLE
AlexNet network

### ISSUE THREAD

**Issue by konglingchoa** [aaron-xichen/pytorch-playground#12]
self.features = nn.Sequential(
            nn.Conv2d(3, 64, kernel_size=11, stride=4, padding=2),
            nn.ReLU(inplace=True),
            nn.MaxPool2d(kernel_size=3, stride=2),
            nn.Conv2d(64, 192, kernel_size=5, padding=2),
            nn.ReLU(inplace=True),
            nn.MaxPool2d(kernel_size=3, stride=2),
            nn.Conv2d(192, 384, kernel_size=3, padding=1),
            nn.ReLU(inplace=True),
            nn.Conv2d(384, 256, kernel_size=3, padding=1),
            nn.ReLU(inplace=True),
            nn.Conv2d(256, 256, kernel_size=3, padding=1),
            nn.ReLU(inplace=True),
            nn.MaxPool2d(kernel_size=3, stride=2),
        )
first code nn.Conv2d(3, 64, kernel_size=11, stride=4, padding=2), is right？
      is not nn.Conv2d(3, 96, kernel_size=11, stride=4, padding=1)？

**Comment by aaron-xichen** [comment]
https://github.com/pytorch/vision/blob/master/torchvision/models/alexnet.py#L19
